File: src/main/java/com/example/springproject/DTO/EmployeeSalaryDTO.java
package com.example.springproject.DTO;

import com.example.springproject.Entities.PaymentStatus;
import lombok.*;

@Setter @Getter @NoArgsConstructor
@AllArgsConstructor
public class EmployeeSalaryDTO {
    private Integer id;
    private String nom;
    private String prenom;
    private byte[] image;
    private float salaireNet;
    private float salaireParJour;
    private int nmbreDeJour;
    private float prime;
    private PaymentStatus paymentStatus;
}

File: src/app/DTO/EmployeeSalaryDTO.ts
import {PaymentStatus} from "../models/employee";

export interface EmployeeSalaryDTO {
  id: number;
  nom: string;
  prenom: string;
  image: string; // On utilisera 'string' pour faciliter l'affichage
  salaireNet: number;
  salaireParJour: number;
  nmbreDeJour: number;
  prime: number;
  paymentStatus: PaymentStatus;
}
